code/game/objects/items/tools/powertool.dm

```diff
@@ -0,0 +1,86 @@
+//Power Tools
+/obj/item/powertool
+	var/stateswap = 0 //what state is it in
+	var/icon_state_swap = 0 //which icon should it be
+	var/tool_behaviour_swap = 0 //what tool behaviour(s) should be here
+	var/swapsound = 0 //the sound that plays for swapping
+
+/obj/item/powertool/attack_self(mob/living/carbon/user) //activating the item causes it to swap icons and behaviours
+	stateswap = !stateswap
+	if(stateswap)
+		icon_state = icon_state_swap
+		tool_behaviour = tool_behaviour_swap
+	else
+		icon_state = initial(icon_state)
+		tool_behaviour = initial(tool_behaviour)
+	playsound(get_turf(user), swapsound, 50, 1)
+	to_chat(user, "<span class='notice'>You attach a different bit to [src].</span>")
+	return
+		
+/obj/item/powertool/drill
+	name = "Power Drill"
+	desc = "A simple hand drill with bolt and screw bits."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "drill_screw"
+	item_state = "drill"
+	icon_state_swap = "drill_bolt"
+	hitsound = 'sound/items/drill_hit.ogg'
+	usesound = 'sound/items/drill_use.ogg'
+	swapsound = 'sound/items/change_drill.ogg'
+	flags = CONDUCT
+	slot_flags = SLOT_BELT
+	force = 5
+	throwforce = 7
```

Comment:
This used to be
```
  force = 8 //might or might not be too high, subject to change
  throwforce = 8
  throw_speed = 2
  throw_range = 3
```
in the screwdriver form and
```
  force = 8 //might or might not be too high, subject to change
  throwforce = 8
  // /obj/item defaults:
  throw_speed = 2
  throw_range  = 7
```
in the wrench form.

Why the change to 5/7 ? Where do those numbers come from?